documentation/src/main/asciidoc/stories/assembly_server_security_credentials.adoc

```diff
@@ -0,0 +1,21 @@
+[id='server_identity']
+:context: server
+= Creating {brandname} Server Credentials
+{brandname} Server allows the encryption of sensitive strings outside of configuration files.
+These strings are stored in a keystore, and subsequently decrypted for applications and verifications systems.
+
+All of the configuration files in `$ISPN_HOME/server/conf/` are world readable by default.
+It is strongly recommended to not store plaintext passwords in the configuration files, and instead place these
```

Comment:
well, you have to store the password for the credential in clear text... not sure how can we solve it.

Reply:
That's unsolveable :)
However this creates an indirection which is sufficient in most cases.